packages/frontend/core/src/modules/workbench/entities/workbench.ts

```diff
@@ -12,12 +15,22 @@ import { View } from './view';
 
 export type WorkbenchPosition = 'beside' | 'active' | 'head' | 'tail' | number;
 
-type WorkbenchOpenOptions = {
+export type WorkbenchOpenOptions = {
   at?: WorkbenchPosition | 'new-tab';
   replaceHistory?: boolean;
   show?: boolean; // only for new tab
+  /**
+   * Whether to keep the search params in the url
+   * if true, keep all search params,
+   * if string[], keep the search params in the array
+   */
+  keepSearch?: boolean | string[];
 };
 
+export interface OpenDocSearchParams {
+  fromTab?: string;
+}
```

Comment:
I think we don't need to define it here? Because this is only a parameter specific to mobile journal, and the complexity should be kept to a minimum.